pkg/watcher/reconciler/dynamic/dynamic.go

```diff
@@ -12,52 +12,60 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package taskrun
+package dynamic
 
 import (
 	"context"
 
-	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
 	"github.com/tektoncd/results/pkg/watcher/reconciler"
 	"github.com/tektoncd/results/pkg/watcher/reconciler/annotation"
 	"github.com/tektoncd/results/pkg/watcher/results"
 	"go.uber.org/zap"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/tools/cache"
 	"knative.dev/pkg/logging"
 )
 
 type Reconciler struct {
-	client            *results.Client
-	pipelineclientset versioned.Interface
-	cfg               *reconciler.Config
+	client    *results.Client
+	gvr       schema.GroupVersionResource
+	clientset dynamic.Interface
+	cfg       *reconciler.Config
+}
+
+type Object interface {
+	metav1.Object
+	schema.ObjectKind
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 	log := logging.FromContext(ctx)
+	log.With(zap.String("resource", r.gvr.String()))
 	log.With(zap.String("key", key))
 
 	namespace, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
 		log.Errorf("invalid resource key: %s", key)
 		return nil
 	}
-
+	log.Infof("key: %s, namespace: %s, name: %s, gvr: %v", key, namespace, name, r.gvr)
 	// Lookup current TaskRun.
```

Comment:
This comment is out of date now.

Reply:
Changed references from TaskRun -> Object.